server/server-utils/emailProvider.js

```diff
@@ -0,0 +1,34 @@
+var nodemailer = require('nodemailer');
+
+function emailProvider() {
+
+  //define environment variables for e-mail
+  if (!process.env.APP_EMAIL || !process.env.APP_EMAILPW) {
```

Comment:
Added check for SMTP. Right now it produces a message if process.env.NODE_ENV is not equal to 'production'

If it IS equal to 'production' it calls process.exit() which should end the process. Unfortunately I can't test this right now as I can't run it in production mode on my system. I think it needs to be on Heroku for that.